Assets/MixedRealityToolkit.Examples/Demos/HandTracking/WebForm/SampleWebForm.htm

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
I wonder if rendering a custom page like this might confuse people. Have you tried instead just launching something like http://aka.ms/mrtk? or http://aka.ms/mrtkdocs?

Reply:
Thanks for reviewing, @julenka . I will delete the web page that demonstrates interacting with a form of buttons/sliders/etc. and instead launch aka.ms/mrdocs in order to demonstrate that you can launch an arbitrary web site. I will also add a button to launch the settings app and make a comment that any 2D UWP application can also be launched in this fashion as documented at https://docs.microsoft.com/en-us/windows/uwp/launch-resume/launch-default-app.

How does that sound?

Reply:
That sounds great, thank you!!!

Reply:
This is implemented in my latest push.